### PR TITLE
Githubfcortez1 Tucson, AZ

### LINKED ISSUE #5: Add a hometown file for Githubteacher
# GitHub Teacher's Hometown
### Why we should do this!

GitHubTeacher would like to get to know the people in class better!

GitHubTeacher travels a lot and loves good recommendations for things to do and places to eat!

@halfaro1 what should I include 🌈 🐶 ?

### PR DESCRIPTION
**Pull request by fcortez1**
Resolves #5 
- [x] create branch
- [x] added file
- [x] committed a file
- [x] opened a pull request 
- [ ] have a discussion
- [ ] make more commits
- [ ] merge pull request

### CODE REVIEW

File: Places/Tucson, AZ [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Tucson, AZ
+
+## Great places to eat
```
Comment (melissaniswonger) [inline Places/Tucson, AZ:3]: Is Char Thai still there? Rosemont and Broadway, i think

File: Places/Tucson, AZ [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+## Fun things to do
+
```
Comment (melissaniswonger) [inline Places/Tucson, AZ:11]: @fcortez1 Do you love the Sonoran Desert Museum? :+1:

File: Places/Tucson, AZ (hunk 1 above)
Comment (fcortez1) [inline Places/Tucson, AZ:3]: @melissaniswonger it is still there!

Comment (melissaniswonger) [inline Places/Tucson, AZ:3]: I loved that place

File: Places/Tucson, AZ [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Tucson, AZ
+
+## Great places to eat
+
+## Fun things to do
```
Comment (melissaniswonger) [inline Places/Tucson, AZ:10]: what about camping in the Chihuahuas?

**Comment by melissaniswonger** [comment]
:shipit:

**Pull request merged**